### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by avneesh0612**
The typos were bugging me so here you go

### CODE REVIEW

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
Most of the corrections are fine, something to keep in mind is that organise vs organize is British vs American, and using one or the other is the author's choice. Don't correct those.

**Comment by avneesh0612** [comment]
Oh I see, I use a vscode extension and it showed me to use the other one 😂. Will remove those

File: portal/guides/create-your-currency-with-python-sdk.mdx [hunk 1]
```diff
@@ -34,7 +34,7 @@ Once your **Token** module has been created, it's time to mint our own currency,
 
 ### 1. Create a Python file and fire up a virtual environment
 
-Go ahead and create a new Python file. Let's call it `currency.py` and open it in your favourite code editor. We'll use Visual Studio Code.
```
Comment (nachoiacovino) [inline portal/guides/create-your-currency-with-python-sdk.mdx]: british english

File: portal/guides/on-demand-minting.mdx [hunk 2]
```diff
@@ -30,7 +30,7 @@ Before we start make sure you have the dependencies installed and understand how
 
 ## What we're going to do
 We're going to build an app that allows users, who have access to this app, to create an unique signature. The signature contains the information of a specific NFT collection that we will pre-define.
-Then with that signature they will be able to mint an NFT from an NFT collection that recognises that signature.
+Then with that signature they will be able to mint an NFT from an NFT collection that recognizes that signature.
```
Comment (nachoiacovino) [inline portal/guides/on-demand-minting.mdx]: british

File: portal/guides/securing-pinning-your-nft-with-ipfs.mdx [hunk 3]
```diff
@@ -13,7 +13,7 @@ updated: ""
 
 ## What is IPFS
 
-When you create an NFT, you need to provide a link to an asset, like an image. That image can be stored on your computer or some server. If the image is gone the NFT is useless. IPFS(inter planetary file system) solves this by saving your images on a network of different nodes. It’s not on the blockchain, but it’s different decentralised network. The image also gains an unique footprint.
+When you create an NFT, you need to provide a link to an asset, like an image. That image can be stored on your computer or some server. If the image is gone the NFT is useless. IPFS(inter planetary file system) solves this by saving your images on a network of different nodes. It’s not on the blockchain, but it’s different decentralized network. The image also gains an unique footprint.
```
Comment (nachoiacovino) [inline portal/guides/securing-pinning-your-nft-with-ipfs.mdx]: british

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
still some british english fixes

**Comment by avneesh0612** [comment]
who decided to create two types of English 🥲😂

**Review by nachoiacovino — APPROVED** [review]
lgtm!

**Pull request merged**